schematic_api/api/routes.py

```diff
@@ -312,16 +313,27 @@ def get_temp_csv(schema_url):
     return tmp_file.name
 
 
+def get_temp_pickle(graph_url):
```

Comment:
Please add in argument types and return types.

Reply:
done!